engine/src/tools/dart/create_updated_flutter_deps.py

```diff
@@ -115,6 +115,13 @@ def Main(argv):
                 updated_value = dart_v.replace(new_vars["dart_git"], "Var('dart_git') + '/")
                 updated_value = updated_value.replace(old_vars["chromium_git"], "Var('chromium_git') + '")
 
+                # We are aiming to produce a quoted string. The code below
```

Comment:
This can be avoided by:
* adding an `'android_git': 'https://android.googlesource.com'` mapping to Flutter's `DEPS`
* changing the `engine/src/flutter/third_party/dart/third_party/perfetto/src` entry in Flutter's `DEPS` to `Var('android_git') + '/platform/external/perfetto@13ce0c9e13b0940d2476cd0cff2301708a9a2e2b'`
* adding `updated_value = updated_value.replace(old_vars["android_git"], "Var('android_git') + '")` above in this script